circe/src/main/scala/io/finch/circe/package.scala

```diff
@@ -4,7 +4,7 @@ import com.twitter.io.Buf
 import io.circe.{Json, Printer}
 import java.nio.charset.{Charset, StandardCharsets}
 
-package object circe extends Encoders with Decoders {
+package object circe extends Encoders with Decoders { self =>
   protected def print(json: Json, cs: Charset): Buf =
     Buf.ByteBuffer.Owned(Printer.noSpaces.prettyByteBuffer(json, cs))
```

Comment:
What if we move this basic implementation under `Encoders` itself and then add overrides when necessary (in each of the configured instances). We wouldn't need the `self.print` redirect in this case.